### ISSUE TITLE
Would like clarification

### ISSUE THREAD

**Issue by David263** [quicbit-js/qb-utf8-from-str-tiny#1]
Could you please add to the readme file an exact description of the arguments and return values for each function? My understanding of this very difficult subject is that there are many ways to express Unicode.

For example, 

array of 32 bit chars: UCS-4
array of 16 bit chars: UCS-2 (BMP only)
array of 16 bits with surrogate pairs: UTF-16
hexadecimal escape sequences
Unicode escape sequences
Unicode code point escapes
code units
JavaScript character strings
HTML character strings

Which ones are supported here? I appreciate the elegant and useful programming, but the documentation is important for us Unicode beginners (I have been a Unicode beginner since it was first announced and I expect to remain a Unicode beginner for many years to come).

Thanks!

**Comment by mvoss9000** [comment]
Good point on the documentation - it is too light. I added doc for the function.  Takes a javascript string and returns an array of integer bytes in UTF-8 format.  Example added as well.
